.github/workflows/docker.yaml

```diff
@@ -33,21 +31,13 @@ jobs:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Check apt updates
-        if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
-        run: |
-            apt-get update -o Dir::Etc::sourcelist="${SOURCELIST}"
-            apt-get --simulate upgrade -o Dir::Etc::sourcelist="${SOURCELIST}" \
-            | grep "^0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.$" \
-            && echo "::set-output name=trigger::false" \
-            || echo "::set-output name=trigger::true"
-            echo "::set-output name=no_cache::true"
```

Comment:
I still want to include this bit but delete no-cache to use your way. Still this is valid to determine if the build should be triggered or not.